src/components/_mixins/MembersListMixin.vue

```diff
@@ -27,6 +27,7 @@ export default {
   },
   mounted() {
     this.fetchMembers(this.groupName);
+    this.membersListOptions = defaultOptions;
```

Comment:
This didn't fix it. I sent you a video on slack. Digging deeper, I think the issue is that `defaultOptions` actually gets changed. To observe just add a `console.log(defaultOptions);` here.

Reply:
Good call. fixed that.